app/src/main/java/me/hufman/androidautoidrive/carapp/carinfo/CarDetailedInfo.kt

```diff
@@ -94,23 +98,9 @@ class CarDetailedInfo(carCapabilities: Map<String, Any?>, cdsMetrics: CDSMetrics
 		} ?: ""
 		"$lat $long${L.CARINFO_GFORCE}"
 	}
-	val gforceLat = cdsMetrics.accel.map { accel ->
```

Comment:
Please don't remove fields, even if they appear unused, because they may be used in the future. People keep asking for customizable screens, and these field variations may be useful for that effort.

Reply:
Restored